### PR TITLE
[BCRYPT_WINETEST] Wine Sync, to let it load on WXP and WS03

### PR DESCRIPTION
**Pull request by SergeGautherie**
## Purpose

This supersedes half of #195.

JIRA issue: [ROSTESTS-296](https://jira.reactos.org/browse/ROSTESTS-296)

## Proposed changes

- Import, as is, the next 2 commits from Wine.

### CODE REVIEW

File: media/doc/README.WINE [hunk 1]
```diff
@@ -49,7 +49,7 @@ reactos/dll/win32/atl                 # Synced to WineStaging-2.9
 reactos/dll/win32/atl80               # Synced to WineStaging-2.9
 reactos/dll/win32/atl100              # Synced to WineStaging-2.9
 reactos/dll/win32/avifil32            # Synced to WineStaging-2.9
-reactos/dll/win32/bcrypt              # Synced to WineStaging-1.9.23
+reactos/dll/win32/bcrypt              # Synced to WineStaging-1.9.23 (+ winetest synced to 8d8936cbb6fea3cac862e059e814527f5361f48b, a.k.a 20161206-BJ)
```
Comment (AmineKhaldi) [inline media/doc/README.WINE]: This will land in the upcoming sync.

Comment (SergeGautherie) [reply media/doc/README.WINE]: I shall wait for that then...

Comment (SergeGautherie) [reply media/doc/README.WINE]: `WineSync` has not happened yet for this file. Could this PR be committed then?

**Comment by ThFabba** [comment]
Not needed. Wine sync will happen when Wine sync happens.

**Comment by SergeGautherie** [comment]
8 months later, `WINESYNC` of this test is still not happening.
Could you re-open, please?

**Comment by SergeGautherie** [comment]
> This will land in the upcoming sync.

Nearly 2 years later, ping @AmineKhaldi

**Comment by JoachimHenze** [comment]
I have not read any justification about why this was closed. @SergeGautherie is this still a problem in master head?

**Comment by SergeGautherie** [comment]
> is this still a problem in master head?

Yes, it still is.

**Comment by GeoB99** [comment]
This PR is pretty much stale.

**Review by JoachimHenze — COMMENTED** [review]
Approved. I will merge this in one week from now unless someone can bring up any reasonable argument about why this would be a bad idea. And no: "Someone else will do the job later" is no reasonable argument, especially not, if the contributer waited for that to happen for 5years.
There are no conflicts, although this PR rottened for 5years. Especially because we did not update this test since at least 2016.
It also passes all the builders. 

I do consider this PR to be a valuable contribution. Not only because it updates the test to a newer state of Wine, than we have atm. But it might contribute towards making more of our crypto-stack run on XP/2k3, which would be a great achievement and very helpful in practice.
On the other hand it also directly contributes ros, because it allows to run the test on our target.

I scheduled PR208 on top of 0.4.15-dev-5076-g26efda4 to run the testbots:
https://build.reactos.org/#/builders/9/builds/32617
VBox https://reactos.org/testman/compare.php?ids=84014,84020 LGTM, all bcrypt tests do still succeed
KVM https://reactos.org/testman/compare.php?ids=84015,84021 LGTM, all bcrypt tests do still succeed

**Review by JoachimHenze — APPROVED** [review]

**Review by AmineKhaldi — APPROVED** [review]
Apologies if this got unintentionally held up!

**Comment by JoachimHenze** [comment]
I tested on XPSP3 patched vs unpatched:
0_before
1_after
It cannot find the bcrypt.dll yet, even if I copy it next to the test, nor when I copy it into Windows/system32.
But we do come further than before.

**Comment by SergeGautherie** [comment]
> cannot find the bcrypt.dll yet

You may be missing other dll(s): see Jira ticket.

**Pull request merged**